### ISSUE TITLE
Cannot filter Events on entity.id

### ISSUE THREAD

**Issue by Dorthu** [linode/linode-cli#106]
The entity.id filter for GET /account/events sends in a string instead of an int:

```
linode-cli events list --entity.type linode --entity.id 12928812
Request failed: 400
┌errors────┬──────────────────────────────┐
│ field    │ reason                       │
├──────────┼──────────────────────────────┤
│ X-Filter │ Entity ID must be an integer │
└──────────┴──────────────────────────────┘
```

**Comment by rcsalome** [comment]
I have a small fix for this. Will make a PR.

**Comment by patthiel** [comment]
This has been shipped with Linode-CLI `v2.10.2`! Closing
